src/components/shared/prompt/prompt.styles.ts

```diff
@@ -16,17 +16,17 @@ export const ScrollableModal = styled(Modal)`
 
 export const GrowContainer = styled.div`
 	margin: auto;
-	padding: 5rem 1.25rem;
+	padding: 5rem ${Spacing["spacing-20"]};
 	width: 100%;
 
 	${MediaQuery.MaxWidth.sm} {
-		padding: 1rem 1.25rem;
+		padding: ${Spacing["spacing-16"]} ${Spacing["spacing-20"]};
```

Comment:
the use of layout here would be more appropriate as this is a full screen container

16px -> layout-sm, 20px -> layout.md